src/main/java/net/neoforged/neoforge/client/event/TextureAtlasStitchedEvent.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.event;
+
+import net.minecraft.client.renderer.texture.TextureAtlas;
+import net.neoforged.bus.api.Event;
+import net.neoforged.bus.api.ICancellableEvent;
+import net.neoforged.fml.LogicalSide;
+import net.neoforged.fml.event.IModBusEvent;
+import net.neoforged.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Fired <b>after</b> a texture atlas is stitched together and all textures therein have been loaded.
+ *
+ * <p>This event is not {@linkplain ICancellableEvent cancellable}, and does not {@linkplain HasResult have a result}.</p>
+ *
+ * <p>This event is fired on the {@linkplain FMLJavaModLoadingContext#getModEventBus()} mod-specific event bus},
+ * only on the {@linkplain LogicalSide#CLIENT logical client}.</p>
```

Comment:
Perhaps we should move this off the mod-specific event buses now, and into the game bus? I believe we're working towards cleaning up events to remove anything not related to mod loading from the mod-specific event buses.

Reply:
Agreed this event has no business being on the mod bus.

Reply:
I don't think the neoforge bus is started early enough for this event. For now everything that happens during the resource reload (e.g. `ModelEvent`) is on the mod bus.

Reply:
@marchermans @sciwhiz12 please see my message. OK for the mod event bus?

Reply:
Hmmmm. I guess we can defer the wholesale moving of events away from the mod bus at a later point, done all at once rather than staggered.